src/components/widgets/EventWidget.tsx

```diff
@@ -42,9 +53,21 @@ export const EventWidget = ({ text }: WidgetProps) => {
     });
   }, [navigation, text, queryVariables]);
 
-  useHomeRefresh(refetch);
+  useHomeRefresh(() => {
+    refetch();
+    showVolunteerEvents && refetchVolunteerEvents();
+  });
+
+  // TODO: filter dataVolunteerEvents by date range already in the request if this will be possible
+  const todaysDataVolunteerEvents = dataVolunteerEvents?.filter(
+    ({ listDate }: { listDate: string }) => listDate === currentDate
+  );
+
+  let eventCount = data?.eventRecords?.length || 0;
 
-  const eventCount = data?.eventRecords?.length;
+  if (showVolunteerEvents && todaysDataVolunteerEvents?.length) {
+    eventCount += todaysDataVolunteerEvents.length;
+  }
```

Comment:
why did you want to increase the `eventCount` here?

Reply:
to add the number of events from volunteer module. otherwise the widget shows a smaller number then the index when clicking on it.

this is what i meant with `add volunteer events count to event widget count` in the commit.